modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -542,13 +542,14 @@ public Account putAccount(Long accountId, Account account)
 		_accountEntryUserRelLocalService.setAccountEntryUserRels(
 			accountId, _getAccountUserAccountIds(account));
 
+		AccountEntry originalAccountEntry =
```

Comment:
save the AccountEntry from line 535 (so I think we don't have to do a fetch again here) and name it just accountEntry

Reply:
then on line 548 you can reassign this to the variable accountEntry